Makefile

```diff
@@ -437,6 +439,20 @@ build-archive:
 release-unix: clean full build-archive
 	@if [ -f e/Makefile ]; then $(MAKE) -C e release; fi
 
+.PHONY:
```

Comment:
Shouldn't `.PHONY` include the target name too? Here and below.

Reply:
If I remember correctly makefiles don't strictly require it (as shown by the `release-unix` target that I didn't modify), but most follow that syntax. However I have updated the targets as requested.